built/History.js

```diff
@@ -4,22 +4,22 @@ var __extends = (this && this.__extends) || function (d, b) {
     function __() { this.constructor = d; }
     d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
 };
-var React = require('react');
```

Comment:
Great, thanks for your contributions!!
I'm still curious as to why there are differences. Perhaps it is the TS compiler?